paperqa/agents/tools.py

```diff
@@ -357,6 +354,8 @@ class Complete(NamedTool):
         r" \| " + EnvironmentState.STATUS_SEARCH_REGEX_PATTERN
     )
 
+    NO_ANSWER_PHRASE: ClassVar[str] = "No answer generated."
```

Comment:
Should we move this to `PQASession` since that's where it will be used most likely? Imo housing this in the `Complete` tool makes it less accessible (since `Complete` tool is sort of an agent implementation detail)